src/components/Header/MenusComp/index.tsx

```diff
@@ -69,14 +69,47 @@ const useMenuDataList = () => {
       name: t('navbar.fee_rate'),
       url: '/fee-rate-tracker',
     },
+    {
+      type: LinkType.Inner,
+      name: t('footer.developer'),
```

Comment:
The label `developer` is not proper IMO, because only `api document` is specific for developers, other contents, e.g. `docs`, `whitepaper`, even `faucet` are also for general users @Danie0918 

Maybe `more` is fine enough for now.

Another consideration is that more sections, e.g. `settings`, and `dashboard` are going to be added. We can re-arrange the menu based on this fact.

I would suggest adding a new top menu where the `language` button is because it's rare for a user to switch languages frequently.

For example, adding a `setting` icon as the top menu, and 
- dashboard
- language
- toolbox
- settings

as sub-menu

Reply:
![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/122437870/17aa1930-de6e-42ba-a5dd-e97ba8000f6c)

Updated, if ok I will push for UI design.

Reply:
cc @Keith-CY

Reply:
UI:https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=36791%3A30145&mode=design&t=V7cY1FZUdWLlWDKj-1 @PainterPuppets

Reply:
Are the changes here going to be done in this pr?

Reply:
> UI:https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=36791%3A30145&mode=design&t=V7cY1FZUdWLlWDKj-1 @PainterPuppets

Is there a change in mobile?

Reply:
> > UI:https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=36791%3A30145&mode=design&t=V7cY1FZUdWLlWDKj-1 @PainterPuppets
> 
> Is there a change in mobile?

![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/122437870/af140c60-d5bf-468a-9044-e2662583b55b)

The existing 'Languages' section can be changed to 'More'.